interface.html

```diff
@@ -1,8 +1,10 @@
 <form class="form-horizontal">
   <div id="testelement" class="hidden" style="background-color: white"></div>
-  <header>
+  <header class="header-block">
```

Comment:
I don't think we need this class, @tonytlwu do you agree?

Reply:
Not unless there are other `header` tags being used